### ISSUE TITLE
Multi columns order Shift + Click on touch devices

### ISSUE THREAD

**Issue by slavap** [DataTables/DataTables#578]
Right now it's impossible to define multi columns order on tablets, because there is no way to make Shift + Click combo. So it would be nice to implement it somehow, for example additional button "multi". Then tap on "multi" and column will activate multi-column ordering.

**Comment by DataTables** [comment]
Sounds like an interesting option. Could probably be done with the new Buttons plug-in that I'm working on for DataTables. Another option might be a multi-touch action.

**Comment by slavap** [comment]
Multi touch won't be convenient, imagine you are holding tablet in left hand and trying to make sorting by the first and last columns using right hand :-)

**Comment by DataTables** [comment]
Fair point. A plug-in for the Buttons extension is probably the way to go then.
